### PR TITLE
Add blob validation

### PR DESCRIPTION
**Pull request by Nepomuceno**
### What does this PR do?

Add validation for blob storage in case there is incomplete data so we do not navidate with incomplete date.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you successfully ran tests with your changes locally? Including container validation.
* [x] Have you lint your code locally prior to submission?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../../pulls) for the same update/change?
* [x] Have you labelled this PR correctly? E.G. major, milestone, breaking, breaking-change, minor, feature, enhancement, patch, fix, chore, refactor etc

### CODE REVIEW

File: client/src/Components/RepoBrowser/AzureBlobBrowser.tsx [hunk 1]
```diff
@@ -28,6 +29,38 @@ const AzureBlobBrowser = () => {
 
   const onSubmit = async (event) => {
     event.preventDefault();
+    if (account === '' || container === '' || sasToken === '') {
+      toast('Please fill in all blob fields', {
```
Comment (sebastus) [inline client/src/Components/RepoBrowser/AzureBlobBrowser.tsx:33]: Please fill in all blob storage account credential fields.

**Review by sebastus — COMMENTED** [review]
little things to clarify

File: client/src/Components/RepoBrowser/AzureBlobBrowser.tsx [hunk 2]
```diff
@@ -28,6 +29,38 @@ const AzureBlobBrowser = () => {
...
+    const todayFormattedDate = todayDate.toISOString().substring(0, 10);
+    const tempDayDifference = Math.abs((Date.parse(todayFormattedDate) - Date.parse(tempExpires)) / 86400000);
+    if (todayFormattedDate > tempExpires) {
+      toast('SAS Token has expired', {
+        icon: '😖',
+        className: 'bg-red-100 font-bold',
+      });
+      return;
+    } else if (tempDayDifference < 7) {
+      toast('Warning this is close to expiration', {
```
[the remark this answers is not in the document]
Comment (Nepomuceno) [reply client/src/Components/RepoBrowser/AzureBlobBrowser.tsx]: Changed

File: client/src/Components/RepoBrowser/AzureBlobBrowser.tsx (hunk 1 above)
Comment (Nepomuceno) [reply client/src/Components/RepoBrowser/AzureBlobBrowser.tsx]: Changed

**Review by sebastus — APPROVED** [review]
lgtm

**Pull request merged**